docs/user-guides/c-types-of-layers/e-polygon.md

```diff
@@ -0,0 +1,20 @@
+# Polygon
+
+Polygon layers can display either paths or polygons in 2D or 3D with geoJSON inputs.
```

Comment:
Polygon layer can display [GeoJSON](https://tools.ietf.org/html/rfc7946)
 features. GeoJSON supports the following geometry types: `Point`, `LineString`, `Polygon`, `MultiPoint`, `MultiLineString`, `MultiPolygon`.

kepler.gl supports all the GeoJSON types above. You can pass in either a single [`Feature`][features] or a [`FeatureCollection`][feature_collection]

[feature_collection]: https://tools.ietf.org/html/rfc7946#section-3.3
[features]: https://tools.ietf.org/html/rfc7946#section-3.2

Reply:
Updated